Multisig/Cross-layer/Analytics/TrackingEvent.swift

```diff
@@ -31,6 +31,8 @@ enum TrackingEvent: String, Trackable {
 
     case assetsNoSafe                               = "screen_assets_no_safe"
     case assetsCoins                                = "screen_assets_coins"
+    case assetsTransferSelect                       =
+        "screen_assets_transfer_select"
```

Comment:
could you please make it one line and add it to this doc https://docs.google.com/spreadsheets/d/1mj9iQIhpM-Pak7lQMb1dmxhQn1QdJoJyhbc730dZNzA/edit#gid=1949479240 ?